src/platforms/react-native/manual-setup/expo.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Expo
+description: "Learn how to set up Expo managed project with Sentry React Native SDK."
+---
+
+<Alert level="info" title="Experimental Support">
+
+The Expo support is currently experimental and available since version `5.16.0-alpha.1` and Expo SDK 50.
+
+Experimental support means it may have bugs. We recognize the irony.
+
+</Alert>
+
+Sentry React Native SDK can be setup in Expo managed project using just one command of `@sentry-wizard`. If you prefer or can't the Wizard you can follow the manual steps on this page.
+
+## Automatic Setup
+
+Run `@sentry/wizard`:
+
+```bash {tabTitle:npm}
+npx @sentry/wizard@latest -i reactNative
+```
+
+## Manual Setup
+
+If you can’t (or prefer not to) run the [automatic setup](/platforms/react-native), use the instructions below to configure your application.
```

Comment:
Should we have the "automatic setup" linked here? In this case, wouldn't the automatic setup be the wizard info right above?

```suggestion
If you can’t (or prefer not to) run the [automatic setup](/platforms/react-native), use the instructions below to configure your application.
```

Reply:
Yes, that's right, I will remove this link.